lib/crp08.py

```diff
@@ -534,7 +545,7 @@ def __str__(self):
 		crp.read_file(sys.argv[3], i)
 		for i in range(1, len(crp.chunks)):
 			bin_file = crp.chunks[0].toc_values[0][i-1]
-			print("-- Extract "+bin_file+" from "+sys.argv[3]+" --")
+			print(f"-- [CHUNK {i}] Extract {bin_file} from {sys.argv[3]} --")
```

Comment:
This is the current (python 3+) format string convention. I find these easier to read.

Reply:
This is nice. I would merge if it's consistent over the whole project.

Reply:
sure. i can do that.

Reply:
I did a big replacement of most string in the project. They are now included here. @Alcantor